.github/workflows/ci.yml

```diff
@@ -11,43 +11,13 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: 12.x
-    - run: yarn install
+    # cache dependencies
+    - uses: c-hive/gha-yarn-cache@v1
+    - name: Install dependencies
+      if: steps.yarn-cache.outputs.cache-hit != 'true'
```

Comment:
What is the purpose of this line? I think it is not necessary after reading this: https://github.com/c-hive/gha-yarn-cache